pro/src/repository/pcapi/pcapi.js

```diff
@@ -360,3 +360,11 @@ export const validateBooking = code => {
 export const invalidateBooking = code => {
   return client.patch(`/v2/bookings/keep/token/${code}`)
 }
+
+//
+// Business Unit
+//
+
+export const getBusinessUnits = offererId => {
+  return client.get(`/offerers/${offererId}/business_unit_list`)
```

Comment:
```suggestion
  return client.get(`/offerers/${offererId}/business_units`)
```
Comme la route back a changé